src/praisonai/praisonai/cli/commands/unified.py

```
@@ -0,0 +1,575 @@
+        return
+    
+    from rich.console import Console
+    console = Console()
+    
+    # Import optional dependencies inside function to avoid startup overhead
+    try:
+        from fastapi import FastAPI, HTTPException
+        from fastapi.responses import HTMLResponse
+        import uvicorn
+    except ImportError as exc:
+        console.print(f"[red]Error: Missing optional dependencies for unified dashboard.[/red]")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)